ocs_ci/ocs/resources/backingstore.py

```diff
@@ -35,6 +36,7 @@ def __init__(
         self,
         name,
         method,
+        type,
```

Comment:
this is a new arg added to this function which does not have a default value. 
Please make sure it doesn't break existing usages and also, please run tier1 for verification

Reply:
It doesn't - the only two are in lines 242 and 270 of this file, and were changed.
I ran `test_bucket_creation` which tests this thoroughly; I'll add a tier1 run soon.